pkg/cli/login/loginoptions.go

```diff
@@ -395,6 +395,7 @@ func (o *LoginOptions) SaveConfig() (bool, error) {
 		return false, err
 	}
 
+	kclientcmd.UseModifyConfigLock = false
```

Comment:
Move this to https://github.com/openshift/oc/blob/master/pkg/cli/shim_kubectl.go#L17 which is invoked for oc only in https://github.com/openshift/oc/blob/master/pkg/cli/cli.go#L347

Reply:
Also it's just one place, which works for all commands.